### ISSUE TITLE
feature need to improve the extendability on the FloatingActionButton

### ISSUE THREAD

**Issue by jjhesk** [cymcsg/UltimateRecyclerView#210]
I think there are requests that this should be improved for other custom appearances.

**Comment by jjhesk** [comment]
Document is [issued in here](https://github.com/cymcsg/UltimateRecyclerView/wiki/Floating-Action-Button-Developer-Guide)

**Comment by chowaikong** [comment]
Yes indeed, the FloatingActionButton of URV can't change icon in code, and there are margin problem of setting ImageRecource.
